jajajaden01/wiredInCodeChallenge1
# setting-up-project

#### What does this PR do?
Wiredin Code Challenge
#### Description of Task to be completed?
- Responsive app as the screenshots appear.
- Usage of the provided assets
- Design eye

- Code structure
- Use of appropriate HTTP verbs and status code.
- Minimum DD traffic as possible
- Better error handling
#### How should this be manually tested?
Make sure that you have cloned this Repo to your local machine
- By running `git clone https://github.com/jajajaden01/wiredInCodeChallenge1.git`
- or download the Zipped folder on `GitLab`
- Then after run `cd wiredInCodeChallenge1` to open the folder or simply double on the downloaded folder
- To install all dependencies locally (on Backend), open the `nodejsBackend` folder and run this command `npm i` or `npm install` in terminal
- To install all dependencies locally (on Frontend), open the `frontend` folder and run this command `yarn install` or `npm i` or `npm install` in terminal
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
- Page of messages with no data
![Screenshot from 2021-02-26 19-53-32](https://user-images.githubusercontent.com/37714031/109340875-425e1900-7872-11eb-98b1-43920e958722.png)

- Page of messages with data
![Screenshot from 2021-02-26 19-54-02](https://user-images.githubusercontent.com/37714031/109340950-599d0680-7872-11eb-8748-a29c02635d65.png)

- Page of Login
![Screenshot from 2021-02-26 19-53-50](https://user-images.githubusercontent.com/37714031/109341000-6ae61300-7872-11eb-8d92-64de3cff5de1.png)

- Page of Register
![Screenshot from 2021-02-26 19-53-41](https://user-images.githubusercontent.com/37714031/109341069-8b15d200-7872-11eb-8731-9828b123d0f8.png)


#### Questions: N/A
